bigtable/export_test.go

```diff
@@ -45,13 +55,25 @@ func init() {
 	flag.StringVar(&c.Project, "it.project", "", "Project to use for integration test")
 	flag.StringVar(&c.Instance, "it.instance", "", "Bigtable instance to use")
 	flag.StringVar(&c.Cluster, "it.cluster", "", "Bigtable cluster to use")
-	flag.StringVar(&c.Table, "it.table", "", "Bigtable table to create")
+	flag.StringVar(&c.Table, "it.table", "it-table", "Bigtable table to create")
 	flag.BoolVar(&c.AttemptDirectPath, "it.attempt-directpath", false, "Attempt DirectPath")
 	flag.BoolVar(&c.DirectPathIPV4Only, "it.directpath-ipv4-only", false, "Run DirectPath on a ipv4-only VM")
 
 	// Backwards compat
 	flag.StringVar(&legacyUseProd, "use_prod", "", `DEPRECATED: if set to "proj,instance,table", run integration test against production`)
 
+	// Don't test instance creation by default, as quota is necessary and aborted tests could strand resources.
```

Comment:
Do we have some way of cleaning up stranded resources after the fact?

Reply:
All tests that create instances are also deferring deletion. that said, they could strand things. There isn't a script to do that in the repo. I have a sh script locally for this since this can be a problem, specifically when debugging (less so in a test run)

Reply:
For storage, we give test buckets a standard prefix, and then when running integration tests there is a method that cleans up any extra buckets in your project with that prefix. Would something like that help here? I think we used the TestMain to accomplish this. https://golang.org/pkg/testing/#hdr-Main

Reply:
Yep. I think this is something we could do. We do want one instance to hang around. The other ones use a standard prefix as well (create-{unix_timestamp}) so we could likely do cleanup. Practically speaking this hasn't been an issue. Though it may be a sane addition.

Reply:
https://github.com/googleapis/google-cloud-go/issues/4157 to track work here.